src/include/uapi/ipc.h

```diff
@@ -382,7 +382,7 @@ struct sof_ipc_dai_dmic_params {
 /* general purpose DAI configuration */
```

Comment:
Your commit message isn't quite right, you are not adding any reserved fields here? Just fixing the enums and revamping the SSP

Reply:
ah, rebase issues now fixing.